File: python/PROBLEMS/1160.py
def anos (pa, pb, ca, cb):
    cont = 0 
    while cont < 100:
        pa+= int(pa*(ca/100))
        pb+=int(pb*(cb/100))
        cont+=1
        if pa>pb:
            print(f"{cont} anos.")
            break
        
        if cont>=100:
            print('Mais de 1 seculo.')
            break

if __name__:
    n = int(input())
    for i in range(0, n):
        pa, pb, ca, cb = list(map(float, input().split(' ')))
        anos(pa,pb,ca,cb)

File: c/PROBLEMS/1160.c
#include <stdio.h>

void anos(int pa, int pb, double ca, double cb){
    int cont = 0;
    while (cont<100){
        pa += (int) pa*(ca/100);
        pb += (int) pb*(cb/100);
        cont++;
        if (pa > pb){
            printf("%d anos.\n", cont);
            break;
        }
        if (cont>=100){
            printf("Mais de 1 seculo.\n");
        }
    }


}

int main(){
    int i, pa, pb, n, a=0;
    double  ca, cb;
    scanf("%d", &n);
    for (i=0;i<n;i++){
        scanf("%d %d %lf %lf", &pa, &pb, &ca, &cb);
        anos(pa, pb, ca, cb);
    }

}